### PR TITLE
Fix slicing of FileManager

### PR DESCRIPTION
**Pull request by Cadair**
Also make movie and quality report optional when downloading from dataset inventory

### CODE REVIEW

File: dkist/io/tests/test_file_manager.py [hunk 1]
```diff
@@ -129,6 +137,18 @@ def test_file_manager_cube_slice(eit_dataset):
...
+    import dkist
+    ds = dkist.Dataset.from_directory("~/dkist_data/BLQRA_fixed/")
+
+    sds = ds[0]
+
+    assert sds.files is not ds.files
+
+    assert len(sds.files) != len(ds.files)
+
+
```
Comment (Cadair) [inline dkist/io/tests/test_file_manager.py:151]:
```suggestion
```

**Comment by Cadair** [comment]
@eigenbrot is this diff sufficient for you?

**Comment by eigenbrot** [comment]
Yep, thanks!

**Comment by Cadair** [comment]
This needs a regression test, so it's a monday job.

**Pull request merged**